index.html

```diff
@@ -0,0 +1,61 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <h3 class="logo">My Logo</h3>
+        
+        <a href="#">
+            <img src="image/hamburger.png" alt="Hamburger" class="hamburger">
+        </a>
```

Comment:
- [ ] As requested by the previous code reviewer, Please make sure that the hamburger button has a `nav` tag. The `nav` tag is for semantics reasons. You can wrap the hamburger like this :point_down: 
  
  ```html
    <nav>
            <a href="#">
                <img src="image/hamburger.png" alt="Hamburger" class="hamburger">
            </a>
    </nav>
  ```